src/clj/harja/palvelin/palvelut/yllapitokohteet/paikkaus.clj

```diff
@@ -23,15 +23,12 @@
   (let [vastaus (into []
                       (comp
                         (map #(konv/string-poluista->keyword % [[:tila] [:paatos]]))
-                        (map #(assoc % :tila (yllapitokohteet-domain/yllapitokohteen-tarkka-tila %)))
-                        (map #(assoc % :tila-kartalla (yllapitokohteet-domain/yllapitokohteen-tila-kartalla %)))
```

Comment:
Tässä oli nyt kaksi ongelmaa: paikkaussilmoituksen tila ylikirjoitettiin kohteen tilaksi. Lisäksi näitä assoccauksia ei enää tarvi tehdä. Nämä tehtiin siksi että päällystysilmoitukset näytettiin kartalla ja nämä tiedot tarvittiin, nyt paikkausilmoitukset-välilehdellä näytetään suoraan kohteet kartalla.